docs/content/installation/integrations/app-protect-waf/compile-waf-policies.md

```diff
@@ -0,0 +1,396 @@
+---
+docs: DOCS-000
+title: Compile NAP WAF policies using NGINX Instance Manager
+weight: 300
+---
+
+## Overview
+
+This guide describes how to use F5 NGINX Instance Manager to compile NGINX App WAF Policies for use with NGINX Ingress Controller.
+
+NGINX App Protect WAF uses policies to configure which security features are enabled or disabled. When these policies are changed, they need to be compiled so that the engine can begin to use them. Compiling policies can take a large amount of time and resources (Depending on the size), so the preferred way to do this is with NGINX Instance Manager, reducing the impact on a NGINX Ingress Controller deployment.
+
+By using NGINX Instance Manager to compile WAF policies, the policy bundle can also be used immediately by NGINX Ingress Controller without reloading.
+
+The following steps describe how to use the NGINX Instance Manager API to create a new security policy, compile a bundle, then add it to NGINX Ingress Controller.
+
+## Before you start
+### Requirements
+- A working [NGINX Management Suite](https://docs.nginx.com/nginx-management-suite/installation/) instance.
+- A NGINX Ingress Controller [deployment with NGINX App Protect WAF]({{< relref "/installation/integrations/app-protect-waf/installation" >}}).
+
+## Configure NGINX Instance Manager
+<!-- This looks like it corresponds to step 1 of the draft:
+
+- The reader will need a working NMS instance (Covered in the requirements above)
+- They will need a user account with API access (Link to NIM/NMS RBAC user creation/auth documentation?) -->
```

Comment:
Unless there's some unique steps to this, a user account with API access could just be part of the pre-requisites - so you jump right into creating a security policy.

Reply:
Basic authentication:

https://docs.nginx.com/nginx-management-suite/admin-guides/authentication/basic-authentication/#making-api-requests-with-basic-authentication

I can't find a simple example of API interactions using OIDC.